templates/custom/harbor-job.yaml

```diff
@@ -0,0 +1,36 @@
+{{- range .Values.projects }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .name }}-job
+spec:
+  template:
+    spec:
+      containers:
+      - name: {{ .name }}
+        image: harbor-helper:3.0
```

Comment:
parameterize this. So that in case we find vulnerabilities during runtime we can swap out to diff image vs having to update the chart